libs/sdk-ui-filters/src/AttributeFilterLoader/impl/multiSelectHandler.ts

```diff
@@ -0,0 +1,55 @@
+// (C) 2022 GoodData Corporation
+import { CallbackRegistration } from "../types/common";
+import { IMultiSelectAttributeFilterHandler, InvertableAttributeElementSelection } from "../types";
+import { AttributeFilterLoader, IAttributeFilterHandlerConfig } from "./loader";
+
+/**
+ * @internal
+ */
+export class AttributeFilterHandler
```

Comment:
Nitpick: the class name does not really match the file name, it is a bit confusing.